metals/src/main/scala/scala/meta/internal/metals/Messages.scala

```diff
@@ -54,7 +54,7 @@ class Messages(icons: Icons) {
     def params: ShowMessageRequestParams = {
       val params = new ShowMessageRequestParams()
       params.setMessage(
-        "New sbt workspace detected, would you like to import the build?"
```

Comment:
Can we keep the build tool name in this message as well?

Also, when running "Import build" for Gradle workspaces the `metals/slowTask` message is "sbt bloopInstall"

<img width="441" alt="Screenshot 2019-05-02 at 19 27 47" src="https://user-images.githubusercontent.com/1408093/57094328-99dfdc00-6d10-11e9-8fab-b95f9c1a9205.png">

Reply:
Will change that, lost that one message